docs/content/preview/migrate/yb-voyager/mysql.md

```diff
@@ -0,0 +1,46 @@
+
+Create a database user and provide the user with READ access to all the resources which need to be migrated. Replace `'localhost'` from the following commands with an appropriate hostname in your setup.
+
+1. Create a new user `ybvoyager`.
+
+   ```sql
+   CREATE USER 'ybvoyager'@'localhost' IDENTIFIED WITH  mysql_native_password BY 'Password#123';
+   ```
+
+1. Grant the global `PROCESS` permission.
```

Comment:
@aishwarya24  Numbering is wrong, `1.` is used for every steps.

Similarly for `oracle.md` and `postgresql.md`

Reply:
ok